### PR TITLE
Update README with correct installation instructions

### LINKED ISSUE #233: installation instructions in README.md are out of date
### What happened?

The posted installation instructions don't work.
I used https://marketplace.upbound.io/providers/crossplane-contrib/provider-kubernetes/v0.13.0 instead.

### What environment did it happen in?
Crossplane version: 1.15.2
Kubernetes: 1.24
Distribution: RKE2

### PR DESCRIPTION
**Pull request by scubbo**
### Description of your changes

Partially addresses #233 

I have:

- [X] Read and followed Crossplane's [contribution process].
- [x] Run `make reviewable test` to ensure this PR is ready for review.
  - Well, I _did_ - but I got ``make: *** No rule to make target `reviewable'.  Stop.``. Similar error for just `make reviewable`.

### How has this code been tested

Confirmed that the new instructions installed a Provider which did not error-out.

[contribution process]: https://git.io/fj2m9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ so using the Crossplane CLI in a Kubernetes cluster where Crossplane is
 installed:
 
 ```console
-kubectl crossplane install provider crossplanecontrib/provider-kubernetes:main
+kubectl crossplane xpkg install provider xpkg.upbound.io/crossplane-contrib/provider-kubernetes:v0.13.0
```
Comment (jbw976) [inline README.md:17]: The [Crossplane CLI](https://docs.crossplane.io/latest/cli/) is no longer a plugin for `kubectl`, so we can remove the `kubectl` from this line as well. 

```suggestion
crossplane xpkg install provider xpkg.upbound.io/crossplane-contrib/provider-kubernetes:v0.13.0
```

**Review by jbw976 — CHANGES_REQUESTED** [review]
awesome @scubbo, thanks for updating these very out of date steps! 😊 

There's more to the readme (e.g. local dev steps) that we'll want to update sometime as well, but this is the critical part to address that is broadly applicable for all users of this provider, so we can follow up in a later PR on the other stuff.

Just one correction to make to remove `kubectl` and then we can merge this - I have tested these updates otherwise and they are working good for me!

**Review by jbw976 — APPROVED** [review]
Looks good to me now, thanks again for making this improvement @scubbo!! 💪

**Comment by scubbo** [comment]
Thanks for the help :)

**Pull request merged**